### ISSUE TITLE
`make clean` fails before `./configure`

### ISSUE THREAD

**Issue by nschloe** [flame/blis#197]
Another one in preparation of a Debian package: `make clean` and friends (`distclean` etc) bail out with
```
common.mk:289: *** Unable to determine compiler vendor. Have you run './configure' yet?.  Stop.
```
They should run without having run configure.

**Comment by devinamatthews** [comment]
For a "normal" autotools project, what is the desired outcome? Without running `configure` I would expect something like

```
[dmatthews@Devin's MacBook Pro tblis]$make clean
make: *** No rule to make target `clean'.  Stop.
```

**Comment by nschloe** [comment]
> make: *** No rule to make target `clean'.  Stop.

Indeed. At some points in the Debian build process, the helpers check if there already is a Makefile and if yes, run `clean` on it. The most autotoolsy approach would probably be to rename `Makefile` to `Makefile.in` and just copy it over at configure time, but perhaps making the `clean` target run before configure is easier and more useful here.

**Comment by devinamatthews** [comment]
Ah, that makes sense. I agree that if `Makefile` exists it should probably work.

**Comment by fgvanzee** [comment]
@nschloe This is hopefully fixed now in b09e4e8. I'll likely move on to issue #195 next.
